README.md

```diff
@@ -1,14 +1,11 @@
 # ipfs-webui - IPFS api frontend
 
-
 ## Dev
 
 ```bash
 git clone https://github.com/ipfs/webui
 cd ipfs-webui
-npm install
-# Runs server on port 8000. Set env PORT to change.
+./alldeps.sh
```

Comment:
why remove the makefile? makefile can do this easier. + can have other targets, like publish.